.github/workflows/publish.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish
+on:
+  release:
+    types: [ published ]
+
+jobs:
+  build-tests:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '3.1.100'
```

Comment:
Any reason this is specifically set to 3.1.100 (and 3.1.x below)?

Reply:
No reason there, this was the specific version which I was working with but it should work for 3.1.x too. I can change it to that 👍